Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -5199,6 +5199,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		61133C772423A4C300786299 /* ⚙️ Run linter */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
This is to fix

> '⚙️ Run linter' will be run during every build because it
does not specify any outputs. To address this warning, either add output dependencies
to the script phase, or configure it to run in every build by unchecking "Based on
dependency analysis"

warning from linter's build phases. Solved by deselecting this option:

<img width="668" alt="Screenshot 2022-12-08 at 10 48 49" src="https://user-images.githubusercontent.com/2358722/206422968-6e56ac7c-79c9-4eb3-aa95-a8c45ca065bc.png">